foundry/foundry.py

```diff
@@ -28,9 +28,8 @@
     FoundryDataset
 )
 
-from foundry.external_data_architectures import (
-    FoundryDataset_Torch
-)
+from foundry.loaders import *
+
```

Comment:
Avoid imports of "*" unless you've got great reason for it. Delaying loading of libraries helps improve responsiveness of Python code, and using `import *` loads everything before you can do any work.